html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>Coding form!</h1>
+  <form>
+    <div class="email">
+      <label for="email-address">Email</label>
+      <div class="email-input">
+        <input type="text" id="email-address">
+      </div>
+    </div>
+    <label for="coding-choices">Why is coding cool?</label>
```

Comment:
The value of a `for` attribute must match the value of the `id` attribute of the form control element this is labelling